pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -301,13 +301,15 @@ rustPlatform.buildRustPackage (finalAttrs: {
     versionCheckHook
   ];
   versionCheckProgram = "${placeholder "out"}/bin/zeditor";
-  versionCheckProgramArg = [ "--version" ];
+  versionCheckProgramArg = "--version";
   doInstallCheck = true;
 
   passthru = {
-    updateScript = gitUpdater {
-      rev-prefix = "v";
-      ignoredVersions = "(*-pre|0.999999.0|0.9999-temporary)";
+    updateScript = nix-update-script {
+      extraArgs = [
+        "--version-regex"
+        "(?!.*(?:-pre|0\.999999\.0|0\.9999-temporary)$)(.+)"
```

Comment:
```suggestion
        "^v(?!.*(?:-pre|0\.999999\.0|0\.9999-temporary)$)(.+)$"
```

Reply:
![grafik](https://github.com/user-attachments/assets/3856b720-ff52-4226-a64c-d00027868d44)

vs

![grafik](https://github.com/user-attachments/assets/418371dc-e90e-4901-86be-929331166b38)

(these are all real tags occuring on the Zed repository)